### PR TITLE
client/tailscale/keys: fix client.Keys unmarshalling

### PR DESCRIPTION
**Pull request by hbowron**
Updates https://github.com/tailscale/tailscale/issues/7020

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203810144085942

### CODE REVIEW

File: client/tailscale/keys.go [hunk 1]
```diff
@@ -13,6 +13,10 @@ import (
 	"time"
 )
 
+type GetKeysResponse struct {
```
Comment (knyar) [inline client/tailscale/keys.go:16]: Could you please make this unexported (`getKeysResponse`) and move it closer to the `Keys` function where it's used (perhaps even inline like the `var keys` was before)?

**Review by knyar — COMMENTED** [review]
Thank you for investigating and fixing this! Looks good, I just have one minor comment.

File: client/tailscale/keys.go (hunk 1 above)
Comment (hbowron) [reply client/tailscale/keys.go:16]: Happy to!  My intention was to keep it consistent with [devices.go](https://github.com/tailscale/tailscale/blob/6edf357b96b28ee1be659a70232c0135b2ffedfd/client/tailscale/devices.go#L21-L23)

Comment (hbowron) [reply client/tailscale/keys.go]: ✅

**Comment by knyar** [comment]
Sorry, I've just noticed that the [DCO check is failing](https://github.com/tailscale/tailscale/pull/7021/checks?check_run_id=10775554935). Could you please add the `Signed-off-by` line to the commit message? It's also probably worth squashing commits into one. Thank you!

**Comment by hbowron** [comment]
> Sorry, I've just noticed that the [DCO check is failing](https://github.com/tailscale/tailscale/pull/7021/checks?check_run_id=10775554935). Could you please add the `Signed-off-by` line to the commit message? It's also probably worth squashing commits into one. Thank you!

@knyar added!  Thanks for all the help btw

**Pull request merged**